src/textual/_node_list.py

```diff
@@ -39,6 +39,20 @@ def __len__(self) -> int:
     def __contains__(self, widget: Widget) -> bool:
         return widget in self._nodes
 
+    def _index(self, widget: Widget) -> int:
```

Comment:
I'd be tempted to make this public. Since this object is already list-like.

Reply:
Will do. For anyone reading: there was a followup chat where we decided that we *do* want the read-only list-a-like methods exposed so they provide some utility when someone does things with `.children`; it's just the methods that modify the `NodeList` that should be treated as private.

Reply:
Done.